### DISCUSSION TITLE
Right click tab menu

### DISCUSSION THREAD

**Discussion by azurtem** [sumatrapdfreader/sumatrapdf#3573] [General]
Hi
A while back you had a version that allowed one to close all opened PDF files to this point, or to the right, etc
It was really convenient, instead of having to manually close each tab 
Will this functionality be reinstated with future versions ?
regards
yann

**Comment by GitHubRulesOK** [comment]
That is currently part of 3.5 Pre-Release (note it is "All Others" or "To Right" NOT to left)
so along with drag and drop tabs "likely" to be included, if its not found too problematic.

**Comment by azurtem** [comment]
Thanks for your response
